pkg/constants/constants.go

```diff
@@ -414,6 +416,16 @@ const (
 	SupportedModelMLFlow      = "mlflow"
 )
 
+// opendatahub rawDeployment Auth
+const (
+	OauthProxyImage                 = "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46"
```

Comment:
Do we need to expose this config? e.g. any updates in the image would require code change in KServe

Reply:
we don't update this oauth proxy image very frequently at all, but do you think I should add it to the configmap and try to consume it from there?

Reply:
Done, kept these constants as fallback values but PR includes changes to the kserve manifests where the oauth proxy params are defined

Reply:
Thanks! LGTM

Reply:
See my other comment: https://github.com/opendatahub-io/kserve/pull/419/files#r1805239200